src/metabase/task/send_pulses.clj

```diff
@@ -43,17 +48,23 @@
                                                     :channel-ids (seq channel-ids)}}
       (log/debugf "Starting Pulse Execution: %d" pulse-id)
       (when-let [pulse (pulse/retrieve-notification pulse-id :archived false)]
-        (metabase.pulse/send-pulse! pulse :channel-ids channel-ids))
-      (log/debugf "Finished Pulse Execution: %d" pulse-id))
+        (metabase.pulse/send-pulse! pulse :channel-ids channel-ids)
+        (log/debugf "Finished Pulse Execution: %d" pulse-id)
```

Comment:
I think this log statement should be outside the `when-let` block, as it was before. Otherwise it can log "Started Pulse Execution: ..." without a corresponding "Finished Pulse Execution: ...", which is confusing